Ladybird/WebContent/main.cpp

```diff
@@ -53,7 +54,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     Web::Platform::ImageCodecPlugin::install(*new Ladybird::ImageCodecPluginLadybird);
 
     Web::Platform::AudioCodecPlugin::install_creation_hook([](auto loader) {
+#if defined(HAVE_PULSEAUDIO)
```

Comment:
hm. Well, if this performs the same/better as the Qt audio codec plugin, then this is probably fine. Andreas asked me to make the Lagom networking a command line switch b/c LibTLS is still missing some popular cipher suites. 

It would be nice to move "use Qt" to an #ifdef in this file rather than a cmdline switch + an ifdef for pulse specifically, but I can mangle that after we get this in.

Reply:
From my testing, the PulseAudio implementation should work better than the Qt plugin did in all cases, so my perspective is that the Qt one should only be used in cases where people can't use the PulseAudio backend. I'm fine with a flag to turn it off, but if any issues occur with it, I'd rather people tell me so I can fix them than switch to the old backend.